app/views/products/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<p id="notice"><%= notice %></p>
+<div id='notice'><%= flash[:notice] %></div>
```

Comment:
notice and alert ko lagi flash[:notice]garnai pardaina, just `notice` gare pugcha